.github/workflows/documentation_accessibility_check.yml

```diff
@@ -0,0 +1,92 @@
+name: '[Documentation] Accessibility Check'
+
+# Adapted from https://docusaurus.io/docs/deployment
+
+on:
+  workflow_dispatch:
+  # Run on Weekdays at 9:00 AM UTC, 4AM ET, 1:00 AM PT
+  schedule:
+    - cron: '0 9 * * 1,2,3,4,5'
+
+jobs:
+  axe-scan:
+    name: Acessibility Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # Step 1: Clean npm cache
```

Comment:
If each step already has a name, these step comments are a bit redundant. Looks good otherwise

Reply:
Yeah, you are right. I removed the steps comments.